crates/cxx-qt-gen/src/lib.rs

```diff
@@ -158,7 +159,8 @@ mod tests {
     ) {
         let parser = Parser::from(syn::parse_str(input).unwrap()).unwrap();
 
-        let generated_cpp = GeneratedCppBlocks::from(&parser).unwrap();
+        let opt = GeneratedOpt::default();
+        let generated_cpp = GeneratedCppBlocks::from(&parser, &opt).unwrap();
```

Comment:
Please add a dummy cfg evaluator here with a few dummy cfg values (e.g. `enabled = true`, `crate = "cxx-qt"`).

Then also please add a new test_input for cfg evaluation that exercises the generation.
This is currently my main point of concern. This change is rather complex, but still lacks high-level snapshot tests.
Having those would greatly increase my confidence in the system.

Reply:
done